scripts/checks.sh

```diff
@@ -167,7 +171,12 @@ function test_balena_registry()
 
 function test_ipv4_stack()
 {
-	# can we reach an IPv4 HTTP service ?
+	# Check if device expects a functioning IPv4 stack
+    if ! ip -4 route get ${IPV4_ADDRESS} &>/dev/null; then
```

Comment:
@20k-ultra indentation?

Reply:
ah...there was a setting in my editor's config that was doing this and I forgot to remove it last time this happened. Resolved.